physionet-django/events/templates/events/email/event_registration.html

```diff
@@ -0,0 +1,13 @@
+{% load i18n %}{% autoescape off %}{% filter wordwrap:70 %}
+Dear {{ name }},
+
+{{ registered_user_name }} has requested to join the Event : {{ event_title }}
+
+You can also use the following link to respond to the request:
+
+{{ url_prefix }}{{ events_home }}
+
+
```

Comment:
Do other emails have two lines before "Regards" or one line?

Reply:
This is interesting. it looks like on `events` app emails we have two lines and on `user` app emails we have single line before `Regards`.

I will add single line for the `event_registration.html` and open another PR for other email templates in events.(unless you recommend i add  one more commit for that here)

Reply:
opened a PR here for extra line https://github.com/MIT-LCP/physionet-build/pull/1897/files